### PR TITLE
Attachments

### PR DESCRIPTION
**Pull request by vladoatanasov**

### CODE REVIEW

File: config.json.template [hunk 1]
```diff
@@ -3,6 +3,13 @@
     "resources" : {
         "resourcesDir": "/path/to/resource-dir",
         "syncUrl": "http://localhost:4985",
-        "bucket": "default"   
-    } 
+        "bucket": "default",
+        "auth" : {
+          "username": "username",
```
Comment (serge-linnet) [inline config.json.template:8]: shouldn't you get it from config file?

File: config_parse.go [hunk 2]
```diff
@@ -8,18 +8,34 @@ import (
...
 //Config represents all resource and sync endpoints
 type Config struct {
 	ResourcesDir string `json:"resourcesDir"`
 	SyncURL      string `json:"syncUrl"`
 	Bucket       string `json:"bucket"`
+	Auth         Auth   `json:"auth"`
 }
 
 //ConfigJSON represents global config properties
 type ConfigJSON struct {
```
Comment (serge-linnet) [inline config_parse.go:27]: Don't you want to keep types in separate files?

File: config.json.template (hunk 1 above)
Comment (vladoatanasov) [inline config.json.template:8]: That's a config file?

File: config_parse.go (hunk 2 above)
Comment (vladoatanasov) [inline config_parse.go:27]: I like the approach where structs, functions and interfaces are grouped in files by feature. Any objections on that?

File: config.json.template (hunk 1 above)
Comment (serge-linnet) [inline config.json.template:8]: Sure... I didn't see the file name :)

File: config_parse.go (hunk 2 above)
Comment (serge-linnet) [inline config_parse.go:27]: Nope, that's clear

**Pull request merged**